website/sidebars.json

```
@@ -18,7 +18,8 @@
       "native-modules-marshalling-data",
       "native-modules-async",
       "native-modules-jsvalue",
-      "native-modules-advanced"
+      "native-modules-advanced",
+      "native-modules-nuget"

```

I think this is supposed to list native-modules-csharp-codegen instead of nuget?
```suggestion
      "native-modules-csharp-codegen"
```